.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/backend"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+- package-ecosystem: npm
+  directory: "/frontend"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+- package-ecosystem: docker
+  directory: "/docker/backend"
+  schedule:
+    interval: weekly
+- package-ecosystem: "github-actions"
+  directory: "/"
```

Comment:
I forgot to ask to set this to `.github/workflows/`. Can you please update it as well?

Reply:
This is what is for actions. By default that is the folder, it looks for. I have already addressed it.